packages/resize/src/index.js

```diff
@@ -0,0 +1,24 @@
+const debounce = (callback, wait) => {
+    let timeout = null
+    return (...args) => {
+        clearTimeout(timeout)
+        timeout = setTimeout(() => {
+            callback.apply(null, args)
+        }, wait)
+    }
+}
```

Comment:
You don't need this part, you can just use `Alpine.debounce` at line 13 to save bytes.

Reply:
Done :)